src/cloud-api-adaptor/pkg/securecomms/sshproxy/sshproxy.go

```diff
@@ -162,6 +186,22 @@ func (inbounds *Inbounds) DelAll() {
 	inbounds.list = [](*Inbound){}
 }
 
+// ParseTag() parses an inbound or outbound tag
+// Outbound tags with structure <Phase>:<Name>:<Port>
+//
+//	are interperted to approach 127.0.0.1:<Port>
+//
+// Outbound tags with structure <Phase>:<Name>:<Host>:<Port>
+//
+//	are interperted to approach <Host>:<Port>
+//
+// Inbound tags with structure <Phase>:<Name>:<Port>
+//
+//	are interperted to serve 127.0.0.1:<Port> on host network namespace
+//
+// Inbound tags with structure <Phase>:<Name>:<Namespace>:<Port>
+//
+//	are interperted to serve 127.0.0.1:<Port> on <Namespace> network namepsace
```

Comment:
Could we have unit tests for these cases as an alternative form of documentation?

Reply:
Tests added for ParseTag and also added a complete SecureComms test with client initiating a connection from a namespace.